docs/distributions/endeavouros/installation.md

```diff
@@ -0,0 +1,63 @@
+# Installing EndeavourOS on a Mac with the T2 Chip
+
+You will need:
+
+- USB drive with at least 4GB
+- A way to plug it into your Mac (USB-C isn't USB-A)
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi.
+
+---
+
+1. Making a partition for Linux.
+
+    1. Open the Bootcamp installer and follow it until it asks for a Windows ISO, this will clear space for a Linux partition (by removing APFS snapshots).
```

Comment:
Will this work if we have Windows installed?

Reply:
This part is copy-paste from the Arch guide. Hmm...

Reply:
I'd prefer using diskutility for partitioning.